kedro-org/kedro-viz
# Chore/change runsList route name to experiment-tracking

## Description

Resolves #819.

## Development notes

This was just a switch of a couple of strings.

## QA notes

Checkout the branch and check the experiment-tracking page.

cc @NeroOkwa take a look at Joel's comment [here](https://github.com/kedro-org/kedro-viz/issues/819#issuecomment-1095272017).

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/820"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
